helm-chart-sources/pulsar/values.yaml

```diff
@@ -1752,6 +1752,29 @@ pulsarHeartbeat:
       topicName: "persistent://public/default/pubsub-latency-test"
   service: {}
 
+# For configuring OpenID in Okta for Pulsar, instructions are here: https://www.youtube.com/watch?v=UQBrecHOXxU&ab_channel=DataStaxDevelopers
+# Then, use the values from the Payload generated to populate these fields:
+openid:
+  enabled: false
+  allowedAudience: api://pulsarClient
```

Comment:
What do you think about using `""` for the default here and then requiring a user to supply the audience? I might need to think about this a bit more.